big_o/complexities.py

```diff
@@ -10,6 +10,7 @@ class NotFittedError(Exception):
 class ComplexityClass(object):
     """ Abstract class that fits complexity classes to timing data.
     """
+    _recalculate_residuals = False
```

Comment:
if we go with this solution, I suggest to rename to `_recalculate_fit_residuals` and add a docstring (a comment formatted with `#: description` above the variable)